tests/settings/test_settings.py

```diff
@@ -18,3 +18,28 @@ def test_invalid_storage() -> None:
         assert False, f"'validate_settings' raised an exception {exc}"
     finally:
         cluster[0]["storage_sets"].remove("non_existing_storage")
+
+
+def test_topics_sync() -> None:
+    from copy import deepcopy
+
+    from snuba.settings.validation import validate_settings
+    from snuba.utils.streams.topics import Topic
+
+    all_settings = {
+        key: value
+        for key, value in settings.__dict__.items()
+        if not key.startswith("__") and not callable(key)
+    }
```

Comment:
All tests on the settings module are likely to have this code. (See the test above). Please put it in a function so that it will be easier to focus on the code you are testing without being side tracked by the boilerplate.